src/helix-ui/mixins/Positionable.js

```diff
@@ -35,10 +35,10 @@ export const Positionable = (superclass) => {
 
             this.$defaultAttribute('position', this.DEFAULT_POSITION);
 
-            this.addEventListener('open', this.__onOpen);
-            this.addEventListener('close', this.__onClose);
-
-            // FIXME: add open listeners if already open
+            if (this.open || !this.open) {
```

Comment:
This condition is redundant. It always returns true.

Instead of wrapping the existing code, why not...

```javascript
this.addEventListener('open', this.__onOpen);
this.addEventListener('close', this.__onClose);

if (this.open) {
  this.__addOpenListeners();
}
```

Reply:
Great tip @CITguy! Changes made.